### ISSUE TITLE
Clarify feedstock LICENSE.txt

### ISSUE THREAD

**Issue by ian-r-rose** [conda-forge/conda-forge.github.io#803]
The docs for contributing and maintaining conda recipes discuss when and how to distribute the LICENSE for a particular package. The auto-generated feedstock repositories also include a license in the root which is different from the related package license. It may be useful to flag this difference in the docs, perhaps [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#post-staging-process)

**Comment by croth1** [comment]
I totally agree we should have this somewhere in the documentation. Maybe we should add a section to the maintainer docs that explain the structure of the feedstock git repositories. This would be a natural place to explain the feedstock license as well.

**Comment by scopatz** [comment]
PRs welcome!

**Comment by PrernaSingh587** [comment]
This issue seems interesting to me. To explain the structure of the feedstock of git repos. If its okay, I would like to take this up.
